subsys/dfu/dfu_target/Kconfig

```diff
@@ -37,6 +37,15 @@ if DFU_TARGET_MCUBOOT_SAVE_PROGRESS
 	comment "DFU_TARGET_MCUBOOT_SAVE_PROGRESS is deprecated, please use DFU_TARGET_STREAM_SAVE_PROGRESS instead"
 endif
 
+if DFU_TARGET_MCUBOOT
+config DFU_TARGET_MCUBOOT_DELIMITER
+	int "Delimiter for MCUBOOT S0 and S1 filenames"
```

Comment:
Any special reason this is an `int ` and not `str` ? I think you could use str. then a `BUILD_ASSERT` to check that the length is no longer than 1

Reply:
Updated as per suggested